kuma/users/templatetags/jinja_helpers.py

```diff
@@ -14,13 +14,13 @@
 
 
 @library.global_function
-def get_avatar_url(user, provider='github'):
+def get_avatar_url(user):
```

Comment:
I guess if the user has both GitHub and Google social accounts, we don't care which avatar we return? Might we want to guarantee the order of the results from the `socialaccount_set` query so it's predictable? For example, perhaps we should order the `socialaccount_set` query by `date_joined` so that we always select the avatar from the first or last social account the user signed-up with?